### PR TITLE
- Updated ios SDK from 16.1 to 16.3.1

### PR DESCRIPTION
**Pull request by rahul-dezerv**
- Upgraded Facebook SDK for iOS from 16.1 to 16.3.1

### CODE REVIEW

**Comment by Saifallak** [comment]
@DennisAlund can you merge this?

**Comment by ventskus-roman** [comment]
@volkankahraman until it's merged, you can use my fork (or create your own)

  facebook_app_events:
    git:
      url: https://github.com/ventskus-roman/flutter_facebook_app_events.git
      ref: ef8b7266b8ab7772159e4703a2d7d1d3addbbbc7

**Comment by volkankahraman** [comment]
@ventskus-roman  Thanks!

**Comment by MaximilianFlechtner** [comment]
+1

File: ios/facebook_app_events.podspec [hunk 1]
```diff
@@ -18,9 +18,9 @@ Flutter plugin for Facebook Analytics and App Events
 
   # Do not specify PATCH version of FBSDKCoreKit. See README file for explanation
   # https://github.com/oddbit/flutter_facebook_app_events#dependencies-on-facebook-sdk
-  s.dependency 'FBSDKCoreKit', '~> 16.1'
+  s.dependency 'FBSDKCoreKit', '~> 16.3'
   
   # See docs on FBAudienceNetwork
   # https://developers.facebook.com/docs/audience-network/setting-up/platform-setup/ios/add-sdk/
-  s.dependency 'FBAudienceNetwork', '~> 6.12'
+  s.dependency 'FBAudienceNetwork', '~> 6.14'
```
Comment (Saifallak) [inline ios/facebook_app_events.podspec:25]:
```suggestion
  s.dependency 'FBAudienceNetwork', '~> 6.15'
```

According to [Meta Docs](https://developers.facebook.com/docs/audience-network/setting-up/platform-setup/ios/add-sdk/)
they released on March 15, 2024 v-6.15

**Review by Saifallak — CHANGES_REQUESTED** [review]

**Comment by antonoddbit** [comment]
I still fixing another branch similar to this, and with less conflict.

**Pull request closed without merging**